lib/net.js

```diff
@@ -2118,19 +2119,38 @@ Server.prototype.listen = function(...args) {
   throw new ERR_INVALID_ARG_VALUE('options', options);
 };
 
+function isIpv6Loopback({ address, family }) {
+  return family === 6 && StringPrototypeStartsWith(address, 'fe80::');
```

Comment:
mmmh thanks for the explaination! learned something new 😄 

I could not find anything which does IPV6 parsing in node, just a couple of regex which are just supposed to validate the addresses and make the distinction between 4 and 6...  it might be there but I wasn't able to find it!

Reply:
I could only find `IN6_IS_ADDR_LINKLOCAL` in the C++ side of things, but nothing on the Js side.  
I have a super simple implementation ready, and battle tested, just waiting for somebody to come up with an existing solution or to tell me to push it 😄  
If so, do you think it would also make sense to expose it, like we do for `isIPv4` and `isIPv6` ?